Makefile

```diff
@@ -549,13 +554,24 @@ docker-build: ## Build the docker image for controller-manager
 docker-push: ## Push the docker image
 	docker push $(CONTROLLER_IMG)-$(ARCH):$(TAG)
 
-.PHONY: build-fkas
+.PHONY: docker-build-fkas
 # Allow overriding this by setting CONTAINER_RUNTIME var
 CONTAINER_RUNTIME := $(if $(CONTAINER_RUNTIME),$(CONTAINER_RUNTIME),docker)
 export CONTAINER_RUNTIME
 
-build-fkas:
-	cd $(FAKE_APISERVER_DIR) && $(CONTAINER_RUNTIME) build --build-arg ARCH=$(ARCH) -t "quay.io/metal3-io/metal3-fkas:latest" .
+docker-build-fkas:
+	cp -r $(FAKE_APISERVER_DIR) /tmp && \
+	mkdir -p /tmp/fake-apiserver/capm3  && \
+	cp -r ./api /tmp/fake-apiserver/capm3 && \
+	cp -r ./baremetal /tmp/fake-apiserver/capm3 && \
+	cd /tmp/fake-apiserver/capm3 && \
+	$(GO) mod init temp/cluster-api-provider-metal3  && \
```

Comment:
When you do this, it defaults the go.mod to the current Go toolchain. In my machine, I'm running Go 1.25.1, which means when the build runs in the Go container, which is 1.24.6, it fails as it doesn't have go 1.25.1 available.